lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -191,6 +191,11 @@
 
         let matches = platform.builtInRegexMatch(locale, LANG_TAG_RE); // String.prototype.match i.e. string.match(re)
         let extensionsString = matches[4];
+        // Note: extensions are /((${extension})-)*/ and are made up of \\b(?:${singleton}(?:-${alphanum}{2,8})+)\\b
+        // where the ${alphanum}{2,8} fields are of the form `${key}-${value}`.
+        // We will return an array of those `${key}-${value}` pairs
+
+        // TODO (doilij): return an array of  `${key}-${value}` pairs
```

Comment:
Resolving this todo will eliminate the need for the ICU special case in `resolveLocaleHelper` in my pull

Reply:
Noted. I'll keep this TODO for future work.